Description: Disables touchpad movements
Ignores touchpad movement history events in order to disable mouse pointer movements with touchpads (prevents accidental mouse pointer movements while typing). Useful for trackpoint devices.
 .
 libinput (1.0.1-1ubuntu1) wily; urgency=medium
 .
   * Merge new upstream release from Debian.
Author: Timo Aaltonen <tjaalton@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- libinput-1.0.1.orig/src/evdev-mt-touchpad.c
+++ libinput-1.0.1/src/evdev-mt-touchpad.c
@@ -917,7 +917,7 @@ tp_process_state(struct tp_dispatch *tp,
 		tp_palm_detect(tp, t, time);
 
 		tp_motion_hysteresis(tp, t);
-		tp_motion_history_push(t);
+		/*tp_motion_history_push(t);*/
 
 		tp_unpin_finger(tp, t);
 
